src/components/vuestic-components/va-option-list/VaOptionList.demo.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+  <VbDemo>
+    <VbCard
+      title="Options as array of string"
+    >
+      <va-option-list
+        :options="baseData.options"
+        :default-value="baseData.options[0]"
+        color="primary"
+        type="radio"
+        v-model="baseData.selected"
+      />
+      Selected {{ baseData.selected }}
+    </VbCard>
+    <VbCard
+      title="Disable all"
+    >
+      <va-option-list
+        :options="baseData.options"
+        type="radio"
+        disabled
+      />
+    </VbCard>
+    <VbCard
+      title="Left label"
+    >
+      <va-option-list
+        :options="baseData.options"
+        type="radio"
+        color="primary"
```

Comment:
You probably want to test colors separately.

Reply:
It's not clear without color prop which radio button is selected. However, I can set `color: primary` as a default prop for OptionList component and show color changing in another card. Would that be ok?